### PR TITLE
solves 4

### PR DESCRIPTION
**Pull request by jchancoding**

### CODE REVIEW

File: curriculum/js1/4.js [hunk 1]
```diff
@@ -6,9 +6,17 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
-};
+const solution = (x, string = "") => {
```
Comment (songz) [inline curriculum/js1/4.js]: your solution should take in 3 parameters. `(x, str, result='')`

**Comment by jchancoding** [comment]
Cleaned up 4 and passes the test now

File: curriculum/js1/4.js [hunk 2]
```diff
@@ -6,9 +6,14 @@
...
  */
 
-const solution = (a, b) => {
-  return 0;
-};
+const solution = (x, str, result="") => {
+  if (!x) {
+    return result;
+  }
+  if (0 < x) {
```
Comment (songz) [inline curriculum/js1/4.js]: don't need a check here.

File: curriculum/js1/4.js [hunk 3]
```diff
@@ -6,9 +6,14 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
-};
+const solution = (x, str, result="") => {
+  if (!x) {
```
Comment (songz) [inline curriculum/js1/4.js:10]: `if(x <= 0)`

**Comment by jchancoding** [comment]
Removed unnecessary if

**Review by songz — APPROVED** [review]

**Pull request merged**